src/nbl/video/IOpenGL_PhysicalDeviceBase.h

```diff
@@ -725,23 +732,39 @@ class IOpenGL_PhysicalDeviceBase : public IOpenGLPhysicalDeviceBase
 			GetIntegerv(GL_MAX_TESS_EVALUATION_SHADER_STORAGE_BLOCKS, maxSSBO + 3);
 			GetIntegerv(GL_MAX_GEOMETRY_SHADER_STORAGE_BLOCKS, maxSSBO + 4);
 			GetIntegerv(GL_MAX_FRAGMENT_SHADER_STORAGE_BLOCKS, maxSSBO + 5);
+			maxComputeShaderResources += maxSSBO[0u];
+			maxVertexShaderResources += maxSSBO[1u];
+			maxTessControlShaderResources += maxSSBO[2u];
+			maxTessEvalShaderResources += maxSSBO[3u];
+			maxGeometryShaderResources += maxSSBO[4u];
+
 			uint32_t maxPerStageSSBOs = static_cast<uint32_t>(*std::min_element(maxSSBO, maxSSBO + 6));
```

Comment:
`maxSSBO[5]` is an uninitialized variable, you'll get random shit here in debug builds or heavy optimized builds depending on compiler

Reply:
The code seems fine to me, line 734 fills maxSSBO+5
Same as the others

Reply:
ah, true, I shouldn't review code at 10pm